app/src/main/java/org/jellyfin/androidtv/util/profile/deviceProfile.kt

```diff
@@ -280,13 +280,49 @@ fun createDeviceProfile(
 		}
 	}
 
-	// Limit video resolution support for older devices
-	codecProfile {
-		type = CodecType.VIDEO
+	// Get max resolutions for common codecs
+	// AVC
+	if(supportsAVC){
+		val maxResolutionAVC = mediaTest.getMaxResolution(MediaFormat.MIMETYPE_VIDEO_AVC)
+		Timber.d("The device supports %s decoding. Max supported resolution: %s", MediaFormat.MIMETYPE_VIDEO_AVC, maxResolutionAVC.toString())
```

Comment:
The createDeviceProfile function should never log, the functions it currently uses (in mediatest) can though.